src/models/weather.js

```diff
@@ -0,0 +1,13 @@
+import Temperature from './weatherData/temperature';
+import Wind from './weatherData/wind';
+import Pressure from './weatherData/pressure';
+
+export default class Weather {
+  constructor(data) {
```

Comment:
I think this name could be more descriptive, data is a very vague term.

Perhaps `inSightWeatherData`? We want to make it clear that our constructors require data from a specific source and won't work with any old data.

Reply:
I agree.